source/agora/flash/Node.d

```diff
@@ -812,9 +941,9 @@ public abstract class FlashNode : ControlFlashAPI
         assert(result.error == ErrorCode.None, result.to!string);
 
         auto channel = new Channel(chan_conf, this.kp, priv_nonce, result.value,
-            peer, this.engine, this.taskman, &this.putTransaction,
```

Comment:
I am not seeing why we need this extra `peer_pk` parameter. Is there something I am missing?

Reply:
Maybe not! I'll try to remove it. I think I needed it briefly during some refactorings.